packages/emails/src/templates/TeamInviteEmail.tsx

```diff
@@ -12,24 +12,30 @@ type TeamInvite = {
   joinLink: string;
   isCalcomMember: boolean;
   isOrg: boolean;
+  parentTeamName: string | undefined;
```

Comment:
```suggestion
  parentTeamName?: string;
```
It makes more sense. As parentTeamName itself isn't undefined but the caller can choose to not pass this.